backend/src/main/java/ca/bc/gov/backendstartapi/endpoint/FavoriteActivityEndpoint.java

```diff
@@ -16,25 +16,19 @@
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 /** This class contains all {@link FavoriteActivityEntity} resources that a user needs. */
 @Setter
 @NoArgsConstructor
 @RestController
-@RequestMapping("/api/favorite_activity")
+@RequestMapping("/api/favorite-activities")
```

Comment:
We also need to change all occurrences of *favorite* and variations to *favourite*, as requested on the design review.